client/src/catalog-community.html

```diff
@@ -259,7 +260,7 @@ <h3 class="page-subtitle">Published on [[_formatDate(_content.metadata.original_
         if (!this._content || !this._content.metadata || !this._content.metadata.link)
```

Comment:
Let's also fire a title change here with the document title.

Some of the /community content is hosted, like the presentations. eg. https://www.webcomponents.org/community/presentations/progressive-performant-polymer-pick-three-at-google-io-2016 so we should update the title there too.